Sources/ExyteChat/Theme/ChatTheme.swift

```diff
@@ -315,3 +321,173 @@ public struct ChatTheme {
         }
     }
 }
+
+// MARK: Accent Color Theme, with default backgrounds
+
+extension View {
+    /// Creates and applies a `ChatTheme` to your `ChatView` based around the provided `accentColor`
+    /// - Parameters:
+    ///   - color: The main accent color of your `ChatView`
+    public func chatTheme(accentColor: Color) -> some View {
+        return self
+            .chatTheme(ChatTheme(accentColor: accentColor))
+    }
+}
+
+extension ChatTheme {
+    internal init(accentColor: Color) {
+        self.init(
+            colors: .init(
+                mainTint: accentColor,
+                messageMyBG: accentColor,
+                messageMyTimeText: Color.white.opacity(0.5),
```

Comment:
why does white come together with accentColor? could you use your accent magic for this too?

also, would you please move all of these t oa separate file like "Theme+Convenience" or smth?

Reply:
The senders text is always `.white`, similar to your default ChatTheme value and Apple's message app. I was adjusting the message bubble color to improve contrast as opposed to altering the text color. I think a pure `.white` text color will always be easier to read over a tinted / off-white text on a colored background. What do you think?

Reply:
I myself do enjoy a white text I must admit) Sure, as long as it wan't a copy/paste oversight